### ISSUE TITLE
Проблема с ответным действием по укд нового форма

### ISSUE THREAD

**Issue by darujo** [diadoc/diadocsdk-1c-docs#698]
Добрый день.
Пытаемся подписать УКД нового формата
1) вызываем по нему метод vReplyTask = iDocument:CreateReplySendTask2("AcceptDocument").
2) в полученном объекте vReplyTask контекст для подписания vContentItems = vReplyTask:ContentItems.
3) и в полученном объекте vContentItems и он не содержит элементов vContentItems:count = 0
4) в следствии чего при вызове метода vReplyTask:send() получаем ошибку 
![image](https://user-images.githubusercontent.com/65738957/135734979-dd6dd10e-6ca9-49c5-919e-81aa6da863ac.png)
Подскажи нам надо по новым документам самим создавать контент для подписания? или мы что-то еще не сделали?

**Comment by darujo** [comment]
OrganizationId VT_BSTR f614e94911d745df8e32c5e0f2e1e1fd@diadoc.ru
DocumentId VT_BSTR 0ef5d66e-72b6-456e-a8b1-dc6b48c853fe3708fb72-9295-4d78-86e9-70bf6d8940eb

**Comment by darujo** [comment]
И точно такая же ситуация , если по Упд запросили корректировку CorrectionRequest
 vReplyTask = iDocument:CreateReplySendTask2("CorrectionRequest").
...
vReplyTask:send() .
То после этого почему-то теперь пописать документ не получается
 vReplyTask = iDocument:CreateReplySendTask2("AcceptDocument").
vContentItems = vReplyTask:ContentItems.
vContentItems не содержит элементов vContentItems:count = 0

Библиотекой пользуемся DiadocCom_5_31_0_604.dll

**Comment by azhipaigor** [comment]
Добрый день.

У вас очень старая версия компоненты, она не поддерживает новые типы документооборота Диадок АПИ ([WorkflowId](https://diadoc-sdk.readthedocs.io/ru/latest/proto/DocumentWorkflow.html) > 11) и соответственно не может определить тип действия при CreateReplySendTask2("AcceptDocument"). Нужно обновить компоненту.

**Comment by darujo** [comment]
Спасибо библиотеку обновили до DiadocCom_5_35_4_736.dll подпись УПД прошла у УКД появился контент.
У УКД при подписи заполняем
Титул:
UniversalCorrectionDocumentBuyerTitle
AcceptanceDate VT_BSTR 05.10.2021
OperationContent VT_BSTR C изменением стоимости согласен
DocumentCreatorBase VT_BSTR 
DocumentCreator VT_BSTR ООО "СИБИРСКАЯ ИНТЕРНЕТ КОМПАНИЯ" (тест), ИНН/КПП 3986228404/964201000
и Подпись
SignerReference
CertificateThumbprint VT_BSTR f4acbfb97c51a6dd138b62272ecaf10c8d2e9374
CertificateBytes VT_BSTR 
BoxId VT_BSTR f614e94911d745df8e32c5e0f2e1e1fd@diadoc.ru
но при отправке vReplyTask:Send() возникает ошибка
![image](https://user-images.githubusercontent.com/65738957/136020414-268c6f4e-7776-46f7-b953-224b4e9e420c.png)
Подскажите какие данные еще необходимо дозаполнить что бы прошла подпись по новому формату УКД. Причем по старым УКД все подписывается.

**Comment by azhipaigor** [comment]
Не заполнена информация подписанта. Можно заполнить методом компоненты [CreateSetExtendedSignerDetailsTask](https://1c-docs.diadoc.ru/ru/latest/ComObjects/Organization.html?highlight=Organization#Organization.CreateSetExtendedSignerDetailsTask) при этом нужно указать тип Ucd736Buyer для DocumentTitleType.

**Comment by darujo** [comment]
Это надо сделать один раз? Если да, то можно эти данные заполнить через веб интерфейс?

**Comment by azhipaigor** [comment]
Да, один раз для отпечатка сертификата. В веб интерфейсе можно заполнить при подписании входящего УКД.

**Comment by darujo** [comment]
А где-то не при подписании УКД можно заполнить данные для подписания. Не очень удобно ждать УКД, чтобы заполнить эти данные через интерфейс? И также я понимаю что это нужно будет делать при каждой смене сертификата?
